packages/tf-next/src/commands/create-deployment.ts

```diff
@@ -73,6 +76,7 @@ async function readConfig(terraformState: any): Promise<CreateDeploymentConfigur
   const snsTopics = jp.query(terraformState, '$..*[?(@.type=="aws_sns_topic" && @.name=="this")]');
```

Comment:
This will not be an issue anymore once we have moved the deployment creation from here into the deployment lambda, because we will pass all this configuration data through environment variables and won't need to parse the TF state anymore.